code/modules/clothing/head/helmets/heavy.dm

```diff
@@ -171,6 +171,22 @@
 	block2add = FOV_BEHIND
 	sellprice = 0 // See above comment
 
+//............... Baothan Helmet ............... //
+
+/obj/item/clothing/head/helmet/heavy/baotha
+	name = "willful helmet"
+	desc = "Baothan knights are an antithesis : the tiefling queen preaches self preservation at the cost of even family or friend. Choosing to embark on a quest to knightlyhood means facing peril at the forefront. \n Indeed, many are the upstarts who understood the folly of their journey, and either turned back to the warmth of ozium and wine or died in the process. \n The original wearer of this helmet is no such pushover : they do not have the might of Graggar nor the magick of Zizo, and instead rely on their wits and grit to proclaim their unicity in the face of this cold, uncaring world. \n It is rumoured that Baotha offers this baroque piece in person, as a reward for the knight's unflinching will in the face of sheer terror, minute after minute, hour after hour. \n Fashioned in steel and petals, it has no viewport, the wearer instead having their senses heightened to preternatural levels." // yes, this is a long item description, but it's the only piece of armour unique to Baotha.
```

Comment:
Comically long item description. I personally think it could be made shorter.

Anyways,
use \ to make the text easier to read when reading the code.
For example;
"This is \\
broken up \\
into three."

Reply:
I'd love to make it shorter, but it's the only Baothan-only piece of armour, and I'd like it to have a bit of a description, to tell the player what this is all about.

Tweaking the code to be more readable though.